apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,262 @@
+`);
+
+const NotificationSettings = ({
+  ignoredEmailNotifications,
+  ignoredInAppNotifications,
+}: NotificationSettingsProps) => {
+  const intl = useIntl();
+  const [, executeMutation] = useMutation(UpdateIgnoredNotifications_Mutation);
+
+  const methods = useForm<FormValues>({
+    defaultValues: dataValuesToFormValues({
+      ignoredEmailNotifications,
+      ignoredInAppNotifications,
+    }),
+  });
+
+  const notificationOptions = (id: string): CheckboxOption[] => [
+    {
+      value: "email",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.email)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+    {
+      value: "inApp",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.inApp)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+  ];
+
+  const { handleSubmit } = methods;
+
+  const handleError = () => {
+    toast.error(
+      intl.formatMessage({
+        defaultMessage: "Error: could not update applicant settings",
+        id: "J1YiBm",
+        description:
+          "Message displayed when an error occurs while updating applicant settings.",
+      }),
+    );
+  };
+
+  const handleFormSubmit: SubmitHandler<FormValues> = async (
+    values: FormValues,
+  ) => {
+    const data = formValuesToData(values);
+    await executeMutation(data)
+      .then((result) => {
+        if (result.data?.updateIgnoredNotifications) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage: "Updated applicant settings",
+              id: "zV8YQ1",
+              description:
+                "Message displayed when applicant updated their settings.",
+            }),
+          );
+        } else {
+          handleError();
+        }
+      })
+      .catch(() => {
+        handleError();
+      });
+  };
+
+  const formFields = [
+    {
+      id: "systemMessages",
+      name: "systemMessages",
+      legend: intl.formatMessage({
+        defaultMessage: "System Messages",
+        id: "KzpzIE",
+        description: "Title for system messages checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "These notifications are required and include email verification and other important communications.",
+        id: "blvpQr",
+        description: "Subtitle for system messages checklist",
+      }),
+      options: notificationOptions("systemMessages"),
+    },
+    {
+      id: "applicationUpdates",
+      name: "applicationUpdates",
+      legend: intl.formatMessage({
+        defaultMessage: "Application updates",
+        id: "v21LPW",
+        description: "Title for application updates checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "Get notified about upcoming application deadlines, status changes, and more.",
```

Comment:
What's the more? It should be explicit what a user is opting in or out of receiving.

Reply:
Again, another design question for @substrae. Maybe @gobyrne can answer this and the question above.

Reply:
This text will change as we evolve our notifications. Leave it as is for now.